frontend/wasm/src/api.rs

```diff
@@ -384,19 +385,19 @@ impl JsEditorHandle {
 
 	/// Cut selected layers
 	pub fn cut(&self) {
-		let message = PortfolioMessage::Cut { clipboard: Clipboard::User };
+		let message = PortfolioMessage::Cut { clipboard: Clipboard::Device };
 		self.dispatch(message);
 	}
 
 	/// Copy selected layers
 	pub fn copy(&self) {
-		let message = PortfolioMessage::Copy { clipboard: Clipboard::User };
+		let message = PortfolioMessage::Copy { clipboard: Clipboard::Device };
 		self.dispatch(message);
 	}
 
-	/// Paste selected layers
-	pub fn paste(&self) {
-		let message = PortfolioMessage::Paste { clipboard: Clipboard::User };
+	/// Paste layers
```

Comment:
Is this comment still accurate? Perhaps if you can describe what "serialized data" means for the informed reader, that would be helpful.